modules/aws/jumpbox/jumpbox.userdata

```diff
@@ -39,7 +39,9 @@ write_files:
         helm pull tetrate-tsb-helm/managementplane --version ${tsb_version} --devel
         tar zxvf managementplane-${tsb_version}.tgz
         for IMAGE in $(cut -d/ -f2- managementplane/images.txt)
-        do 
+        do
+          IMGNAME=$(echo $IMAGE | cut -d: -f1)
+          aws ecr create-repository --repository-name ${registry_name}/$IMGNAME --region ${region} || true
```

Comment:
Repositories need to exist before pushing. this is something `tctl image sync` does internally, but when manually copying needs to be done explicitly. The `|| true` is to avoid having the command fail if the repositories exist (on eventual Jumbox recreations).